test/e2e/cypress/integration/hosts_overview.js

```diff
@@ -104,12 +109,14 @@ context('Hosts Overview', () => {
             ]
             healthScenarios.forEach(([health, expectedHostsWithThisHealth], index) => {
                 it(`should show ${expectedHostsWithThisHealth || 'an empty list of'} hosts when filtering by health '${health}'`, () => {
-                    cy.intercept('GET', `/hosts?per_page=100&health=${health}`).as('filterByHealthStatus') 
+                    cy.intercept('GET', `/hosts?per_page=100&health=${health}`).as('filterByHealthStatus')
                     const selectedOption = `#bs-select-1-${index}`
                     cy.get(selectedOption).click()
                     cy.wait('@filterByHealthStatus').then(() => {
                         expectedHostsWithThisHealth == 0 && cy.get('.table.eos-table').contains('There are currently no records to be shown')
                         expectedHostsWithThisHealth > 0 && cy.get('.tn-hostname').its('length').should('eq', expectedHostsWithThisHealth)
+                        cy.get('.pagination-count').should('contain', `${expectedHostsWithThisHealth} items`)
```

Comment:
As a suggestion for the future:

This could be a custom `cy` command https://docs.cypress.io/api/cypress-api/custom-commands, and we can think of it as a custom assertion for pagination.

Something along these lines:
```
Cypress.Commands.add('paginationShouldMatch', (itemsCount) => {
    cy.get('.pagination-count').should('contain', `${itemsCount} items`)
    cy.get('.page-item').its('length').should('eq', Math.ceil(itemsCount/100)+2)
})
```

which then becomes accessible as
```
cy.paginationShouldMatch(expectedHostsWithThisHealth)

cy.paginationShouldMatch(expectedRelatedHosts)

cy.paginationShouldMatch(expectedTaggedHosts)
```
in the different variants.

Look, don't bother changing it now, as I understand it wouldn't add much value right now :smile: 
Just wanted to share the cypress concept of custom command that might become more and more useful as we go.